src/convertToObject.js

```diff
@@ -1,12 +1,38 @@
 'use strict';
 
+const complexStylesString = require('./stylesString');
+
 /**
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const splited = sourceString.split(';');
+
+  const clear = splited.map((elem) => {
+    return elem.trim();
+  });
+
+  const filtred = clear.filter((elem) => {
+    return elem.length > 0;
+  });
```

Comment:
The variable name `filtred` is a typo. It should be `filtered` to correctly represent the past tense of 'filter'.